### PR TITLE
[feat]: Adding Navbar CSS

### PR DESCRIPTION
**Pull request by raghav2110**
Hi I have created navigation bar. Please review my PR.
1. Added Dark Navbar
2. Added Light Navbar
3. Added Navbar With Search And Icons

Screenshot 2022-02-07 at 12 50 25 AM
Screenshot 2022-02-07 at 12 50 57 AM
Screenshot 2022-02-07 at 12 51 05 AM

### CODE REVIEW

**Review by junaidshaikh-js — COMMENTED** [review]
Looks good to me!

File: components/navbar/navbar.css [hunk 1]
```diff
@@ -28,6 +28,20 @@
...
 }
 
+.nav .search-container{
+    display: flex;
+    align-items: center;
+}
+
+.nav input{
+    padding: 0.4rem;
+    margin-top: 0.5rem;
```
Comment (hinanshisuthar) [inline components/navbar/navbar.css:38]: Preffered to use px rather than rems for values <=10.

**Review by hinanshisuthar — COMMENTED** [review]
Everything else looks good to merge.

File: components/navbar/navbar.css (hunk 1 above)
Comment (raghav2110) [reply components/navbar/navbar.css:38]: Okay noted.

**Pull request merged**